mmv1/products/alloydb/Cluster.yaml

```diff
@@ -452,3 +460,22 @@ properties:
       - !ruby/object:Api::Type::String
         name: 'sourceType'
         description: 'Type of migration source.'
+  - !ruby/object:Api::Type::Enum
+    name: clusterType
+    values:
+      - :PRIMARY
+      - :SECONDARY
+    default_value: :PRIMARY
+    description: |
+      The type of cluster. If not set, defaults to PRIMARY.
+  - !ruby/object:Api::Type::NestedObject
+    name: "secondaryConfig"
```

Comment:
We should be able to add a conflict between this and automatedBackupPolicy

Reply:
One can add automatedBackupPolicy with automatedBackupPolicy.enabled = false. So can not use conflict for this case